makefiles/murdock.inc.mk

```diff
@@ -0,0 +1,35 @@
+#
+# This file contains helper targets used by the CI.
+#
+
+# (HACK) get actual flash binary from FFLAGS.
+FLASHFILE:=$(filter $(HEXFILE) $(ELFFILE:.elf=.bin) $(ELFFILE),$(FFLAGS))
+
+#
+# This target will run "make test" on the CI cluster.
+#
+# In order to work, these requirements must be fulfilled:
+# - DWQ_REPO and DWQ_COMMIT are set correctly
+# - the user has set up autossh & proper authentication for connecting to the CI
+# (intended to be used by CI only for now)
+test-murdock:
+	cd $(RIOTBASE) && \
+		./.murdock test_job \
+		$$(realpath --relative-to $(RIOTBASE) $(APPDIR)) \
+		$(BOARD) \
+		$(FLASHFILE)
+
+# don't whitelist tests if there's no binary
+ifeq (1,$(RIOTNOLINK))
+  TEST_WHITELIST:=
```

Comment:
Maybe call this variable TEST_BOARD_WHITELIST to be more explicit ?

Reply:
how about TEST_ON_CI_WHITELIST?

Reply:
CI_BOARD_WHITELIST ?

Reply:
are boards in CI_BOARDS_WHITELIST compile-tested or not?

Reply:
> are boards in CI_BOARDS_WHITELIST compile-tested or not?

Good point. Since you need to build before running, there's no need for it indeed. The HIL test will also check the build.

Reply:
So TEST_ON_CI_WHITELIST is better than TEST_WHITELIST? @aabadie @cladmi?

Reply:
I even missed it was a variable only for CI and not for tests in general. So yes having it explicit is good.